packages/component-library-design-tokens/src/lint-build.mjs

```diff
@@ -0,0 +1,81 @@
+import { readFile } from 'node:fs/promises';
+import cloneDeepWith from 'lodash.clonedeepwith';
+import isPlainObject from 'lodash.isplainobject';
+import { glob } from 'glob';
+import { getComponentTokens } from './lib.mjs';
+import { isDesignTokenDefinition } from '@nl-design-system-unstable/theme-toolkit/dist/design-tokens.js';
+
+export const getCss = async () => {
```

Comment:
Zijn al (of de meeste van) deze functies duplicaat met `packages/component-library-design-tokens/src/build-tokens.mjs`? Is het dan niet handiger om ze op 1 plek te zetten?